daemon/volumes.go

```diff
@@ -138,6 +139,14 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		if label.RelabelNeeded(bind.Mode) {
+			if _, err := os.Stat(bind.Source); err != nil {
```

Comment:
Ah, yes, tripped over that one (also thought the stat was redundant), so I added that comment.
If the same comment applies here; could you add that comment here as well, to prevent others from tripping over it?